usr/sbin/p11sak/p11sak.c

```diff
@@ -2604,8 +2602,10 @@ static CK_RV list_ckey(CK_SESSION_HANDLE session, CK_SLOT_ID slot,
         rc = tok_key_get_key_type(session, hkey, &keyclass, &keytype,
                 &keylength);
         if (rc != CKR_OK) {
-            fprintf(stderr, "Invalid key type (error code 0x%lX: %s)\n", rc,
-                    p11_get_ckr(rc));
+            if (rc != CKR_KEY_TYPE_INCONSISTENT)
+                fprintf(stderr,
+                        "Retrieval of key type failed (error code 0x%lX: %s)\n",
+                        rc, p11_get_ckr(rc));
```

Comment:
We could print out the related key type in the error msg to give a hint into what the user's input translated and caused the error

Reply:
I don't know the key type here. The error CKR_KEY_TYPE_INCONSISTENT is just returned as a hint by tok_key_get_key_type that its not a key object at all.
We could choose another error code if that fits better....

Reply:
Ah, I see .... I thought this is the p11sak_kt. The error code is fine with me.